packages/contracts-bedrock/src/libraries/Predeploys.sol

```diff
@@ -103,7 +103,7 @@ library Predeploys {
         if (_addr == OPTIMISM_MINTABLE_ERC20_FACTORY) return "OptimismMintableERC20Factory";
         if (_addr == L1_BLOCK_NUMBER) return "L1BlockNumber";
         if (_addr == L2_ERC721_BRIDGE) return "L2ERC721Bridge";
-        if (_addr == L1_BLOCK_ATTRIBUTES) return "L1Block";
+        if (_addr == L1_BLOCK_ATTRIBUTES) return _useInterop ? "L1BlockInterop" : "L1Block";
```

Comment:
I think its ok to remove this diff and just call it `L1Block` even tho its `L1BlockInterop` - eventually we will merge `L1BlockInterop` into `L1Block`

Reply:
We can use the semver to identify if the code has the feature